module-utils/time/test/unittest_time.cpp

```diff
@@ -519,6 +519,19 @@ TEST_CASE("Duration - creation")
         REQUIRE(duration.get() == 333);
     }
 
+    SECTION("std::chrono::seconds constructor")
+    {
+        utils::time::Duration duration(std::chrono::seconds(333));
+        REQUIRE(duration.get() == 333);
```

Comment:
What's the deal with these methods? Doesn't `std::chrono` provide this functionality already?